src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -54,6 +55,10 @@ def __init__(self, instrument, *args, **kwargs):
         self._default_nplc = 10  # Default NPLC setting as per Fluke 8846A manual
         self._init_string = ""  # Unchanging
 
+        # High and low current port definition. Each definition encodes the maximum current able to
+        # be measured by the port (in amps)
+        self.current_ports = {"HIGH": 10, "LOW": 400e-3}
```

Comment:
I think `self.current_ports` should be private in the hopes that people aren't tempted to update them.